tests/spread/smoketests/metafiles/unenforced_bad_yaml/task.yaml

```diff
@@ -0,0 +1,98 @@
+summary: >
```

Comment:
these tests are not running, the structure is

    suite/task

You have

    suite/suite/task

Here's the result

```
$ spread -list google: | grep smoketests
google:ubuntu-18.04-64:tests/spread/smoketests/basic-centos-7
google:ubuntu-20.04-64:tests/spread/smoketests/basic
google:ubuntu-20.04-64:tests/spread/smoketests/basic-non-lts
google:ubuntu-20.04-64:tests/spread/smoketests/destructive-mode
google:ubuntu-20.04-64:tests/spread/smoketests/different-dependencies
google:ubuntu-20.04-64:tests/spread/smoketests/different-dir
google:ubuntu-20.04-64:tests/spread/smoketests/full-lifecycle
google:ubuntu-20.04-64:tests/spread/smoketests/pinned-dependencies
google:ubuntu-20.04-64:tests/spread/smoketests/reactive
google:ubuntu-22.04-64:tests/spread/smoketests/basic
google:ubuntu-22.04-64:tests/spread/smoketests/basic-almalinux
google:ubuntu-22.04-64:tests/spread/smoketests/basic-non-lts
google:ubuntu-22.04-64:tests/spread/smoketests/destructive-mode
google:ubuntu-22.04-64:tests/spread/smoketests/different-dependencies
google:ubuntu-22.04-64:tests/spread/smoketests/different-dir
google:ubuntu-22.04-64:tests/spread/smoketests/full-lifecycle
google:ubuntu-22.04-64:tests/spread/smoketests/pinned-dependencies
google:ubuntu-22.04-64:tests/spread/smoketests/reactive
```

Please flatten it to

    - tests/spread/smoketests/metafiles-bad-yaml
    - ...